src/oneD/MultiNewton.cpp

```diff
@@ -204,14 +204,14 @@ int MultiNewton::dampStep(const double* x0, const double* step0,
                           OneDim& r, MultiJac& jac, int loglevel, bool writetitle)
 {
     // write header
-    if (loglevel > 0 && writetitle) {
-        writelog("\n\nDamped Newton iteration:\n");
-        writeline('-', 65, false);
-
-        writelog("\n{}  {:>9s}   {:>9s}     {:>9s}   {:>9s}   {:>9s}  {:>5s} {:>5s}\n",
-                "m","F_damp","F_bound","log10(ss)",
-                "log10(s0)","log10(s1)","N_jac","Age");
-        writeline('-', 65);
+    if (loglevel > 0) {
+        writelog("\n\n{}\n", "Damped Newton iteration:");
+        writeline('-', 83, false);
+
+        writelog("\n{:<6s}  {:<9s}    {:<9s}      {:<9s}   {:<9s}   {:<9s}  {:<5s}  {:<5s}\n",
+                 "Iter", "F_damp", "F_bound", "log10(ss)",
+                "log10(s0)", "log10(s1)", "N_jac", "Age");
+        writeline('-', 83, false);
```

Comment:
The thing that I was trying to keep straight is at high levels, we have loops:

steadySolve->MultiNewton::solve() -> MultiNewton::dampStep() -> MultiNewton::boundStep()

unsteadySolve->OneDim::timeStep() ->MultiNewton::solve()-> MultiNewton::dampStep() -> MultiNewton::boundStep()

And the logging output within the timestepping loop is where things get dicey because there's a ton of output within a single timestep attempt and that's what I was trying to make clearer.

Reply:
I think I cleaned it up with the newest commit. Hopefully more readable and less cluttered.